README.md

```diff
@@ -69,11 +69,11 @@ Either the name of the `entity` or:
 | `entity`              |                   | The entity id                                                                                 |
 | `display`             | `marker`          | Either `icon` or `marker`. `marker` will display the picture if available                     |
 | `size`                | 24                | Size of the icon (not supported for `marker`)                                                 |
-| `history_start`       |                   | Either a `date (__2022-03-01T12:00:00Z__)` or a `relative timestamp (__5 hours ago__)`        |
-| `history_end`         |                   | Either a `date (__2022-03-01T12:00:00Z__)` or a `relative timestamp (__now__)`                |
+| `history_start`       |                   | Examples: `2022-03-01T12:00:00Z`, `5 hours ago`                                               |
+| `history_end`         |                   | Examples: `2022-03-01T18:00:00Z`, `2 hours ago`, `now`                                        |
 | `history_line_color`  | Random Color      |                                                                                               |
 
-If `history_start` is set it will display a line with all the previous locations, as does the `hours_to_show` functionality does. This map does only render the entity specified (the HA map renders also the source `device_tracker`).
+If `history_start` & `history_end` are set it will display a line with all the previous locations (like with `hours_to_show` functionality for a stock HA map card). This card does only render the entity specified (`hours_to_show` affects all entities in the stock HA map card).
```

Comment:
Add a clarification that for example `person.john` in stock HA will likely pull in the history for it's associated device_trackers (eg. `device_tracker.john_phone`).

Reply:
Stock HA map shows a path for at least `device_tracker`, `person`, `sensor` domains - if an entity has `latitude` & `longitude` attributes. That is why I thought that mentioning a particular case "person is based on device_tracker" could be not critical.
Besides, as far I I believe, when placing a `person` entity on a stock map card - data are taken from DB for this `person` entity, not for some `device_tracker` (btw, there could be several device_trackers associated with a person). So, that old statements can be considered even as a bit confusing (((((

Reply:
Yes, however I noticed if I take a person entity I don't see all the points, while if I directly use the associated device_tracker it does. 

Maybe just this as a cleaned up sentence will avoid extra confusion.

Reply:
> Yes, however I noticed if I take a person entity I don't see all the points, while if I directly use the associated device_tracker it does.

Hmmm, hard to tell. Could this case:
- person has 3 trackers - 1 router-based & 2 gps-based (assume - Life360 & Traccar);
- when a person is at home - a router-based tracker gets a priority; a person entity has no coords (and hence not shown on a map);
- when a person not home - a gps-tracker with the latest update gets a priority; it may cause a "flickering" between two zones located nearby since each tracker feeds different coords (same physical device - but apps are working differently); as a result - a person gets a "combined" history (partly from Life360, partly from Traccar).

This is as per Docs & own experience.
But in fact it may work differently; there are plenty of bugs in HA...

> Maybe just this as a cleaned up sentence will avoid extra confusion.

What if we remove everything about the stock map card?
Btw a behaviour of the stock card may change - for instance, a `hours_to_show` could be replaced by a "time period" object (there are rumours from Devs).

Reply:
Thx for the investigation!

Let's indeed keep it simple, we'll see if we get extra questions

Reply:
Edited a text a bit - please have a look.